api/tests/src/db/models/factories.py

```diff
@@ -234,6 +236,27 @@ class Meta:
         sqlalchemy_session_persistence = "commit"
 
 
+class OpportunityAttachmentFactory(BaseFactory):
+    class Meta:
+        model = opportunity_models.OpportunityAttachment
+
+    attachment_id = factory.Sequence(lambda n: n)
+
+    file_location = factory.Faker("url")
+    mime_type = factory.Faker("mime_type")
+    file_name = factory.Faker("file_name")
+    file_description = factory.Faker("sentence")
+    file_size_bytes = factory.Faker("random_int", min=1000, max=10000000)
+    opportunity_attachment_type = factory.Faker(
+        "random_element", elements=OpportunityAttachmentType
+    )
```

Comment:
Another way you can do this - slightly less to type.
```suggestion
    opportunity_attachment_type = factory.fuzzy.FuzzyChoice(OpportunityAttachmentType)
```